### ISSUE TITLE
[Sourcing-2][Accessibility: ACC-264.1] pop up: Incorrect UI Type read.

### ISSUE THREAD

**Issue by JyotiKumari1507** [SAP/fundamental-ngx#11273]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
Component having issue: https://sap.github.io/fundamental-ngx/#/platform/table/basic
Table with Sort:
‘Sort Ascending’  and 'Sort descending' UI are read out as ‘Link’ and the filter list item is not provided with any Role. Type of the UI should be read out correctly to screen reader users.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All latest version

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Using Screen reader can check by implementing similar case. Just use table with sort on toolbar. ‘Sort Ascending’  and 'Sort descending' UI are read out by screen reader as ‘Link’ and the filter list item is not provided with any Role.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations. 
Yes we have got this bug from Accessibility expert only.

#### Did you check the documentation and the API?
Yes

**Comment by nikolay-kolarov** [comment]
Hi @droshev,

This is about the the items in the popup here:
![image](https://github.com/SAP/fundamental-ngx/assets/17043004/f9c68602-d2b5-400b-83ba-f1785ab227d9)
In some cases the focus goes over the listitems, in other over anchors. So this needs to be checked. Also, I am not sure why you need anchors here.
The implementation in UI5 uses menu with menuitems: https://ui5.sap.com/#/entity/sap.ui.table.Table/sample/sap.ui.table.sample.Sorting

Best Regards,
Nikolay
